### PR TITLE
Fix bug #84 and #89

### PR DESCRIPTION
**Pull request by olivier-stasse**
Fix bug #84 related to addBounds for UnitaryOp VectorSelecter / Selec_of_vector.
Fix bug #89 on checking position when no integration is performed.
Add unittest on addBounds and other unitary operators.

### CODE REVIEW

File: src/tools/device.cpp [hunk 1]
```diff
@@ -527,7 +523,6 @@ void Device::integrate( const double & dt )
   {
     assert(state_.size()==controlIN.size()+6);
     state_.tail(controlIN.size()) = controlIN;
-    CHECK_BOUNDS(state_, lowerPosition_, upperPosition_, "position");
```
Comment (jmirabel) [inline src/tools/device.cpp]: What is the reason for removing this ?

File: src/tools/device.cpp [hunk 2]
```diff
@@ -254,10 +254,6 @@ setState( const Vector& st )
           throw std::invalid_argument ("Upper and/or lower velocity bounds "
               "do not match state size. Set them first with setVelocityBounds");
       case CONTROL_INPUT_NO_INTEGRATION:
-        if (   s != lowerPosition_.size()
-            || s != upperPosition_.size() )
-          throw std::invalid_argument ("Upper and/or lower position bounds "
-              "do not match state size. Set them first with setPositionBounds");
```
Comment (jmirabel) [inline src/tools/device.cpp]: What is the reason for removing this ?

File: src/tools/device.cpp (hunk 1 above)
Comment (olivier-stasse) [reply src/tools/device.cpp]: When no integration is done, the state_ is not making sense and specifically when using torque control. The motor control is current and this means that we should have an actuator model to make a correct integration. This is not yet available.

File: src/tools/device.cpp (hunk 2 above)
Comment (olivier-stasse) [reply src/tools/device.cpp]: Same reason than above.

**Pull request merged**